scenes/game/game.tscn

```diff
@@ -347,7 +349,7 @@ flip = true
 [node name="PlayerDeck" type="Node2D" parent="."]
 
 [node name="PlayerCharacterCard" parent="PlayerDeck" instance=ExtResource("18_imast")]
-position = Vector2(1058, 574)
+position = Vector2(1056, 576)
```

Comment:
Did you mean to adjust this stuff by a couple pixels?
If not, revert these changes.

Reply:
intentional